### DISCUSSION TITLE
Welcome to Publii Discussions!

### DISCUSSION THREAD

**Discussion by GetPublii** [GetPublii/Publii#1331] [Announcements]
## 👋 Welcome!
Hello, and welcome to our discussion forum! We’re switching the forum from our custom solution to GitHub Discussion. We think this change will bring many benefits for you and us. But don’t worry; the content from the [old forum](https://forum.getpublii.com/) won’t be lost. It will still be there in read-only mode so that you can access previous discussions and materials.

Github Discussion is a new feature from Github that lets you create and manage discussions related to open-source projects. It’s a great place for our community because:

* It makes it easy to communicate with you and collect your feedback, suggestions, and bug reports. 
* It lets you find and join interesting threads, ask questions, and get answers from other users and developers. 
* It integrates with other GitHub tools, such as Issues, Pull Requests, and Actions, which help us develop and maintain our project. 
* It provides a high level of security and privacy, as it uses the same authentication and authorization system as Github. 
* It will fix many issues that users reported, such as code formatting and adding threads to favorites. We’re making these improvements to make the forum more comfortable and functional for all community members.

To join the new forum, [create a GitHub account](https://github.com/join) (if you don’t already have one) and visit our Github Discussion page. Please note that moving to Github Discussion means we’ll have all new discussions there. The [old forum](https://forum.getpublii.com/) will remain in read-only format, ensuring existing content is preserved but not moved to Github.

We hope you’ll love our new forum platform and that you’ll use it actively. We’re thankful for your support and engagement in our project. Please contact us if you have any questions or concerns about moving to Github Discussion.

**Comment by internettips** [comment]
Cool: looking forward to it. Thank you for all you folks do with Publii.

**Comment by candidexmedia** [comment]
Thank you Publii team! Although the previous forum was incredibly useful, there were a [number of limitations](https://forum.getpublii.com/topic/forum-limitations/), and a solution within your existing bug tracking tool makes a lot of sense. GitHub Discussions is a great choice!

I just wish an email had been sent to all forum account holders to give us a heads up before closing it (ex: closing registrations first, then disallowing the ability to create new threads, then adding a warning for new replies). I was in the forum a few days ago and didn't know it was migrating.

**Reply by bobmitro** [reply]
The old forum is still open for current users; you still have access to your posts and data. However, it is no longer possible to create new threads there, as a new tool has arrived.  I believe you will experience a positive change.

**Reply by candidexmedia** [reply]
Hi @bobmitro! I definitely appreciate that the previous posts and data are still available in read-only mode. I've bookmarked many important forum posts over the years that contained useful tips for reference. I'm grateful that the wealth of community knowledge shared over the years will be preserved (on the Publii site or Internet Archive).

That said, I still believe that a phased closure and advance notice by email would have been ideal. Many threads and conversations were still ongoing, and our ability to comment has been removed overnight.

There's no way for me to alert participants of an existing thread that I created a continuation post on GitHub Discussions so that we can continue discussing. I just have to wait for them to notice the banner on the forum site, and hope they find the thread again. If I knew a migration was coming (with, say, a 30 or 90 day warning), I could have let them know.

**Reply by bobmitro** [reply]
> There's no way for me to alert participants of an existing thread that I created a continuation post on GitHub Discussions so that we can continue discussing.

Good point of view. I have re-enabled the feature to reply to previously shared topics for 30 days.

**Reply by candidexmedia** [reply]
Thank you @bobmitro !
